warning.c

```diff
@@ -0,0 +1,15 @@
+int warning(float value,float minlimit,float maxlimit)
```

Comment:
Use an enum to convey the meaning of return value, rather than native type like int